### ISSUE TITLE
Create QnA Maker knowledge base by importing URL issue

### ISSUE THREAD

**Issue by zxyanliu** [microsoft/BotFramework-Composer#5162]
## Describe the bug

Was unable to create a QnA Maker knowledge base by importing URL.

## To Reproduce

1. Create QnA KB from template
2. Select QnA from Composer menu
3. Select Create new KB
4. Enter KB name and URL 
5. Select View KB and URL was not loaded.
6. Enter QnA KB authoring key and Start the bot. The KB can be published to QnA portal with success, but the content showed completely different as the imported URL should.
7. Test in Emulator with example question cannot get the expected answer. 

## Screenshots

![image](https://user-images.githubusercontent.com/32497439/101226302-df7d2d00-3648-11eb-8dfb-42038112af4f.png)

![image](https://user-images.githubusercontent.com/32497439/101227771-e8bcc880-364d-11eb-8bb4-7532d636deea.png)

![image](https://user-images.githubusercontent.com/32497439/101226843-a34acc00-364a-11eb-9b87-2415ce66441c.png)

## Additional texts
- Testing URL: https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/troubleshooting
- It might be an issue from the QnA side, which needs investigation.

**Comment by zhixzhan** [comment]
Hi @zxyanliu, extracting QnA from url requires good HTML document format, if url source is web page.

I noticed the URL https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/troubleshooting
has some embed tab, even show/hide by JS code which is not be able to extract in `bf-lu` parser.

This whole page is extracted to one QnA pair: 
<img width="980" alt="image" src="https://user-images.githubusercontent.com/49866537/101324350-1485bb80-38a5-11eb-9be2-5259c38954a0.png">

if you ask "Troubleshooting for QnA Maker", it would answer with: 
<img width="698" alt="image" src="https://user-images.githubusercontent.com/49866537/101324462-4434c380-38a5-11eb-99b5-dd1b7c61598e.png">

that is expected.

**Comment by zxyanliu** [comment]
Hi @zhixzhan Thanks for your response.
FYI. This is the URL we provided in the documentation and it worked for 1.2.1 and previous versions.
https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/troubleshooting

If this URL was not supported, can you please provide specifications of what type of URL are supported? Can you elaborate a bit more on "it must a static HTML page with good semantic format. e.g."?

**Comment by zxyanliu** [comment]
Hi @zhixzhan 
Yes, it is the issue with the testing URL. I tried this troubleshooting URL in QnA portal separately and failed to load the questions properly. I will use another URL such as https://download.microsoft.com/download/2/9/B/29B20383-302C-4517-A006-B0186F04BE28/surface-pro-4-user-guide-EN.pdf in the docs. Closing this issue for now because it is not a Composer issue after initial investigation.
